cursorless-talon/src/modifiers/containing_scope.py

```diff
@@ -69,6 +69,7 @@ def cursorless_containing_scope(m) -> dict[str, dict[str, Any]]:
     "file": "document",
     "line": "line",
     "paint": "nonWhitespaceSequence",
+    "small paint": "nonwhiteSpaceOrDelimiterSequence",
```

Comment:
Heyo @AndrewDant — I'm wondering if we should consider moving this definition to [selection_type.py](https://github.com/cursorless-dev/cursorless/blob/main/cursorless-talon/src/modifiers/selection_type.py#L9) rather than defining a separate selection type. It seems to me like `small` is a modifier on `paint` and we implement an optional preceding parameter in the `selection_type` file. 

We could then send a different sub type of `SelectionType` into the TS side and map the regex that way. 

This is my preference but I'm also not up to speed on the fully compositional modifiers changes. 

I'm definitely down to talk about it, just let me know. Also interested to hear others' opinions.

Reply:
I think I'd prefer it as is. I'll prob remap it to a single word, which I couldn't do if it were a modifier on "paint"

Reply:
💯  got it — I wanted to get some thoughts down to understand where we might differ.